### PR TITLE
reduce with to drag events on the same time

### PR DESCRIPTION
**Pull request by Julian-B90**
Fix this issue https://github.com/mattlewis92/angular-calendar/issues/1447

### CODE REVIEW

File: src/calendar-utils.ts [hunk 1]
```diff
@@ -679,7 +679,7 @@ function getWeekViewHourGrid(
         )
       );
 
-      const width = 100 / columnCount;
+      const width = 90 / columnCount;
       return { ...event, left: event.left * width, width };
```
Comment (mattlewis92) [inline src/calendar-utils.ts:683]: This is technically a breaking change so can we make it an option instead that gets passed in through the outer functions and default it to 100 to preserve backwards compatibility.

**Review by mattlewis92 — CHANGES_REQUESTED** [review]
Nice one, thanks for this! Can we make it an option though and pass it through via an input from the day + week view components of the calendar?

File: src/calendar-utils.ts (hunk 1 above)
Comment (Julian-B90) [reply src/calendar-utils.ts:683]: sure

**Comment by Julian-B90** [comment]
@mattlewis92 i have add columnWidth as variable for the width.

**Comment by mattlewis92** [comment]
I had a think about this some more and realised you can achieve this with CSS, if you add this to your global styles it should do the trick:

```
.cal-week-view .cal-time-events .cal-events-container {
  width: 90%;
}
```

Let me know if that works for you 😄

**Comment by Julian-B90** [comment]
Hey @mattlewis92 yes the css code works. Thank you :-)

**Comment by mattlewis92** [comment]
OK great! I'm gonna close this out one then, whenever possible I tend to prefer CSS fixes for stuff like this, otherwise the API becomes massive with tonnes of options 😄

**Pull request closed without merging**